src/main/java/core/basesyntax/converter/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.converter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(filePath))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file at " + filePath, e);

```

Comment:
Good job on rethrowing a RuntimeException with an informative message and exception object. This is a good practice in handling exceptions.